Makefile

```diff
@@ -1,4 +1,6 @@
-all:
+all: build runserver
```

Comment:
Could you please remove `runserver` from this target list? I just wanted `make` to build the server itself, plus any other plugins, docs, etc. but not run the server.